examples/observability_archive/version.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.0.0"
+  required_providers {
+    # Pin to the lowest provider version of the range defined in the main module to ensure lowest version still works
+    ibm = {
+      source  = "ibm-cloud/ibm"
+      version = "1.49.0"
+    }
+    logdna = {
+      source  = "logdna/logdna"
+      version = "1.13.1"
```

Comment:
will need to be locked into 1.14.2 after updating in version.tf

Reply:
Updated